components/Cms/Element/CmsElementImageSlider.vue

```diff
@@ -106,12 +106,22 @@ if (isHomePage.value) {
                     />
                 </LayoutSliderSlide>
             </LayoutSlider>
-        </div>
 
-        <template v-else>
-            <div class="w-full bg-gray-light">
-                <SharedImagePlaceholder :size="'lg'" />
-            </div>
-        </template>
-    </ClientOnly>
+            <template #fallback>
+                <img
+                    v-cms-element-lazy-load="{ id: slides[0].media.id, type: 'image' }"
+                    :src="slides[0].mediaUrl"
```

Comment:
```suggestion
                    v-cms-element-lazy-load="{ id: slides?.at(0)?.media?.id, type: 'image' }"
                    :src="slides?.at(0)?.mediaUrl"
```

maybe we shoudl also add a v-if

Reply:
The v-if is already added because we check the slides?.length in a parent div